src/main/java/gov/usds/case_issues/authorization/Http401UnauthorizedEntryPoint.java

```diff
@@ -0,0 +1,39 @@
+package gov.usds.case_issues.authorization;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.json.JSONObject;
+import org.json.JSONException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.AuthenticationEntryPoint;
+
+public class Http401UnauthorizedEntryPoint implements AuthenticationEntryPoint {
+
+	private static final Logger LOG = LoggerFactory.getLogger(CustomAccessDeniedHandler.class);
+
+	@Override
+	public void commence(HttpServletRequest request, HttpServletResponse response,
+		AuthenticationException authException) throws IOException, ServletException {
+			String loginURL = "/oauth2/authorization/uscis-icam";
```

Comment:
I think `clientLogin` would be better for the front-end since it takes the `redirect` parameter, which would let the front-end be able to dictate where the user is sent back to.

Reply:
:+1: